_episodes/38.md

```diff
@@ -0,0 +1,509 @@
+---
+layout: episode
+title:  "38: Trino tacks on polymorphic table functions"
+date: 2022-07-21
+tags: trino polymorphic table functions
+youtube_id: ""
+wistia_id: ""
+sections:
+- title: "Releases 387 to 390"
+  desc: "Details about features and fixes in the new releases"
+  time:
+---
+
+## Guests
+
+In this episode we have the pleasure to chat with a couple familiar faces who
+have been hard at work building and understanding the features we're talking
+about today:
+
+### [Kasia Findeisen](https://github.com/kasiafi), Trino Maintainer
+
+[TODO - Kasia]:<>
+
+### [Martin Traverso](https://twitter.com/mtraverso), Trino Co-Creator & Maintainer
+
+[TODO - Martin]:<>
```

Comment:
@martint if there's anything you want to add here, please feel free to suggest it.